app/controllers/admin/orders_controller.rb

```diff
@@ -1,4 +1,29 @@
 class Admin::OrdersController < Admin::BaseController
+
   def index
+    # user = User.find_by(username: params[:session][:username])
+    # if user.current_admin
+    # @orders = Order.all
+    @orders = Order.all
+    # byebug
+      # byebug
+    # else
+    #   render:
+    # end
+    @status = Order.group(:status).count
+    byebug
   end
+
+  def sort_orders
+    # status = params[:order][:status]
+      @orders = Order.get_by_status(params[:order][:status])
+      @status =Order.all.group(:status).count
+      # byebug
+      render 'admin/orders/index'
```

Comment:
instead of redirecting back to the dashboard it just re-renders with updated instance variables